docs/reference/migration/migrate_7_0/settings.asciidoc

```diff
@@ -43,11 +45,13 @@ of the node id. It can still be configured explicitly in `elasticsearch.yml`.
 
 [float]
 [[remove-http-enabled]]
+//tag::notable-breaking-changes[]
```

Comment:
I'm wondering how many users are disabling HTTP, should we keep it a non-notable breaking change?